advanced/helm/is-with-analytics/is-with-analytics/templates/identity-server-analytics-dashboard-deployment.yaml

```diff
@@ -20,23 +20,45 @@ metadata:
   labels:
     deployment: wso2is-with-analytics-is-analytics-dashboard
 spec:
-  replicas: 1
+  replicas: {{ default 1 .Values.wso2.deployment.wso2isAnalyticsDashboard.replicas }}
   strategy:
     rollingUpdate:
-      maxSurge: 1
-      maxUnavailable: 0
+      maxSurge: {{ default 1 .Values.wso2.deployment.wso2isAnalyticsDashboard.strategy.rollingUpdate.maxSurge }}
+      maxUnavailable: {{ default 0 .Values.wso2.deployment.wso2isAnalyticsDashboard.strategy.rollingUpdate.maxUnavailable }}
     type: RollingUpdate
   template:
```

Comment:
@Savidude @ThilinaManamgoda we may have to add resource limits and requests for each WSO2 product profile. Please see the following example resource limits and requests definition for WSO2 API Manager deployment patterns.

```
          resources:
            requests:
              memory: 2Gi
              cpu: 2000m
            limits:
              memory: 3Gi
              cpu: 3000m
```
@ThilinaManamgoda Have we added these to Helm resources for deployment pattern 1 of WSO2 IAM?

Reply:
No, we didn't add them. We have to add this

Reply:
Added https://github.com/wso2/kubernetes-is/pull/145/commits/609fa9de7b8eafcdfd810404dbd0c17c28580d46